pyintegration/deephaven/csv.py

```diff
@@ -15,12 +15,17 @@
 _JCsvTools = jpy.get_type("io.deephaven.csv.CsvTools")
 _JParsers = jpy.get_type("io.deephaven.csv.parsers.Parsers")
 _JArrays = jpy.get_type("java.util.Arrays")
-
+_JQueryConstants = jpy.get_type("io.deephaven.util.QueryConstants")
```

Comment:
duplicate work with the constants module, should use 'from deephaven.constants import MAX_LONG'